model/Core/Datatypes/MediaType.md

```diff
@@ -4,19 +4,20 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-Standardized way of indicating the type of content of an Element. A String constrained to the RFC 2046 specification.
+Standardized way of indicating the type of content of an Element. A String constrained to the RFC 6838 specification.
 
 ## Description
 
-A MediaType is a string constrained to the RFC 2046 specification. It provides a standardized
-way of indicating the type of content of an Element.
+A MediaType is a string constrained to the RFC 6838 specification, and includes
+`type-name`, `subtype-name` and optional `parameters`.
+It provides a standardized way of indicating the type of content of an Element.
 A list of all possible media types is available at https://www.iana.org/assignments/media-types/media-types.xhtml.
 
 ## Metadata
 
 - name: MediaType
+- SubclassOf: xsd:string
 
 ## Format
 
-- pattern: ^[^\/]+\/[^\/]+$
-
+- pattern: ^([a-zA-Z0-9][-a-zA-Z0-9!#$&^_.+]{0,126})\/([a-zA-Z0-9][-a-zA-Z0-9!#$&^_.+]{0,126})(;.+)?$
```

Comment:
I'd expect the type (left) part to be 

    application|audio|font|example|image|message|model|multipart|text|video

Reply:
The RFC does not say that - the ABNF allows for additional registries and doesn't restrict their names.  But SPDX could choose to allow just the existing registries.

Reply:
@goneall we already mention that the list of accepted types are the ones in the IANA list. Why allow arbitrary strings?